rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/SemanticTokenizer.java

```diff
@@ -74,24 +71,12 @@ public SemanticTokenizer(boolean rascal) {
         this.patch = rascal ? new RascalCategoryPatch() : new DefaultCategoryPatch();
     }
 
-    @Override
-    public SemanticTokens semanticTokensFull(ITree tree) {
+    public SemanticTokens semanticTokensFull(ITree tree, boolean specialCaseHighlighting) {
         TokenList tokens = new TokenList();
-        new TokenCollector(tokens, patch).collect(tree);
+        new TokenCollector(tokens, specialCaseHighlighting, patch).collect(tree);
         return new SemanticTokens(tokens.getTheList());
     }
 
-    @Override
-    public Either<SemanticTokens, SemanticTokensDelta> semanticTokensFullDelta(String previousId, ITree tree) {
-        return Either.forLeft(semanticTokensFull(tree));
-    }
-
-    @Override
-    public SemanticTokens semanticTokensRange(Range range, ITree tree) {
-        return semanticTokensFull(tree);
-    }
```

Comment:
Unused functions (but I still needed to update them because the signature of `semanticTokensFull` changed). I thought I'd just remove them instead... If we need them in the future, we can add them again.